great-idea/services.html

```diff
@@ -15,93 +15,122 @@
+          eleifend. Mauris euismod facilisis iaculis.
+          </p> 
+          <button>Learn More</button> 
 
+      </dev>
+      <dev class="text-content">
+          <h1>Web Development</h1>
 
-UX / UI
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.
+        </p> 
+          <button>Learn More</button> 
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+      </dev>
 
-Learn More
+      <dev class="text-content">
+          <h1>iOS Development</h1>  
 
+          <p> Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.
+          </p>
 
+          <button>Learn More</button> 
 
-Digital Marketing
+      </dev>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+      <dev class="text-content">
+          <h1>Android Development</h1>
 
-Learn More
+         <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.
+        </p> 
+          <button>Learn More</button> 
+      </dev>
+    </dev>
+</section>
+<br>
+<br>
+<br>
+<br>
+<section>
 
+<dev class="last">
 
+  <dev class="crt">
 
-Web Development
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+  <h1>Some Facts About Our Services</h1>
 
-Learn More
+  <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+  eleifend. Mauris euismod facilisis iaculis.
+  </p> 
+      <li>Awesome thing</li>
```

Comment:
a `<ul></ul>` surrounding your list items might help here. Then you can position the whole list targeting the `<ul>`.